<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>phishincidenteditor</class>
 <widget class="QDialog" name="phishincidenteditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2d42a8;color:white;</string>
  </property>
  <widget class="QTextEdit" name="incidentAnalysis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>1201</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Enter Analysis/Timeline/Response to Incident:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>1201</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Enter Malicious Senders Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Enter Email Subject:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Did the email have attachments?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Enter SHA256 hash of files attached:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Enter IP Address of Malicious Sender:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Enter Date and Time of Incident:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Enter Recipient of Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="malSender">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>741</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailSubject">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>741</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailAttach">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>741</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailSHA">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>741</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maliciousIP">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>741</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailRecipient">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>741</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="emailDateTime">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>741</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Enter Severity:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="severity">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Low</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Ticket Status:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ticketStatus">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Open</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>In Progress</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Closed</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>1020</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Phishing Incident Creation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>1040</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
